README.md

```diff
@@ -1,3 +1,11 @@
+# Math Magicians
+
+Have your own calculator verson on your local machine. Responsive on all screens. Built to suit your needs.
+
+## Live Demo
+
+Not available yet 
```

Comment:
- [ ] Please use [this](https://github.com/microverseinc/readme-template/blob/master/README.md) readme template provided by Microverse and edit each section to better suit your project.